doc/radosgw/metrics.rst

```diff
@@ -117,11 +98,51 @@ For a user ``User1`` doing operations on ``Bucket1``, the RGW Op Metrics can be
         ...
     ]
 
+Enabling the Radosgw User & Bucket Counter Caches
+=================================================
+
+To track op metrics by user the Radosgw the config value ``rgw_user_counters_cache`` must be set to ``true``. 
+
+To track op metrics by bucket the Radosgw the config value ``rgw_bucket_counters_cache`` must be set to ``true``. 
+
+These config values are set in the Ceph configuration file under the ``[client.rgw.{instance-name}]`` section and must be set before the radosgw is started or restarted to take effect.
+
+The counters in these caches live in memory. If the radosgw is restarted or crashes, all the  counters in the cache are lost.
+
+Radosgw User & Bucket Counter Cache Size & Eviction
+===================================================
+
+The user and bucket counter cache sizes should be sized appropriately depending on the expected number of users and buckets in the cluster.
```

Comment:
is this really what we want to recommend? if users configure these caches to cover *all* of their users and buckets, why bother with a cache or eviction in the first place?

i thought these cache sizes were more about limiting how much we dump into prometheus, rather than just as limits to memory usage